src/main/java/com/internet/shop/model/Order.java

```diff
@@ -6,12 +6,17 @@
     private Long id;
     private List<Product> products;
     private Long userId;
+    private Double price;
```

Comment:
please remove this field
we should not store this in Order - we will have trouble when we will connect to the DB